metronome/checkpointing/interpreter/src/io/iohk/metronome/checkpointing/interpreter/InterpreterRPC.scala

```diff
@@ -26,7 +26,7 @@ trait InterpreterRPC[F[_]] {
   def createBlockBody(
       ledger: Ledger,
       mempool: Seq[Transaction.ProposerBlock]
-  ): F[Option[Block.Body]]
+  ): F[Option[(Block.Body, Set[Transaction.ProposerBlock])]]
```

Comment:
I wouldn't mind a doc string explaining the return type, as on its own it's not very clear

Reply:
Yeah, I was wondering if I should introduce a custom case class or at least a type alias for it, because it got tiring to type it in.  Just when I typed in `BlockBodyWithPurgeFromMempool` I thought that's a mouthful. I'll have another go at it.

Reply:
Added the type alias and some helper functions, thanks for calling this out.